object/asset.go

```diff
@@ -138,6 +138,13 @@ func getAsset(owner string, name string) (*Asset, error) {
 	}
 
 	if existed {
+		vaultClient, err := util.InitVaultClient()
```

Comment:
Why do you handle get? why not connect (via RDP/SSH, etc.)?

Reply:
I had a discussion with Supervisor Yu. He added the mask in the asset.go file within the controller. So, I first decrypt the data, then add the mask. The password will not be exposed to the front-end.